core-web/apps/dotcms-ui/src/app/portlets/dot-containers/container-list/container-list.component.html

```diff
@@ -1,4 +1,81 @@
 <dot-portlet-base>
     <dot-portlet-toolbar title="Containers"></dot-portlet-toolbar>
-    <p>container-list works!</p>
+
+    <dot-listing-data-table
+        #listing
+        [actionHeaderOptions]="actionHeaderOptions"
+        [columns]="tableColumns"
+        url="v1/containers"
+        [actions]="[]"
+        [checkbox]="true"
+        dataKey="inode"
+    >
```

Comment:
I think we can set `actions` as an empty array on the `DotListingDataTableComponent` component by default. That way we avoid passing an empty array when there are no `actions`.